DOL.WHD.Section14c.Web/src/modules/components/mainNavigationControl/mainNavigationControlController.js

```diff
@@ -13,6 +13,15 @@ module.exports = function(ngModule) {
     'use strict';
 
     var vm = this;
+    vm.sections = navService.getSections().map(function(element){
```

Comment:
Can this be replaced with `vm.sections = this.getSections()`?

Reply:
@mgwalker Let's ask Harun, I believe this was his change. The changes from his associated chore are part of this PR. (I think this was for chore #306) @davoodharun  can you comment? Thanks.

Reply:
I removed this.getSections. It was not needed.